### PR TITLE
Add Main and README to Examples

### PR DESCRIPTION
**Pull request by doriac11**
# Description
- Add Main and README to Examples

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- NA this is a doc update

### CODE REVIEW

**Comment by doriac11** [comment]
run e2e test

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!--
+Copyright (c) 2023-2024 Dell Inc., or its subsidiaries. All Rights Reserved.
```
Comment (taohe1012) [inline examples/README.md:2]: hi @doriac11 , the cp years will use 2023-2024 ?

Comment (doriac11) [reply examples/README.md:2]: Ahh you are right it should just be 2024, I will update this.

**Comment by doriac11** [comment]
run e2e test

File: examples/main.tf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+terraform {
+  required_providers {
+    powerflex = {
+      source  = "registry.terraform.io/dell/powerflex"
```
Comment (gokul-srivathsan) [inline examples/main.tf:21]: Why this file now? should we add version of this provider here?

Comment (doriac11) [reply examples/main.tf:21]: I was doing what Powerscale did also, no version needed since that is subject to change.

**Review by gokul-srivathsan — APPROVED** [review]
LGTM

**Pull request merged**